packages/service-clients/azure-client/api-report/azure-client.api.md

```diff
@@ -68,7 +68,7 @@ export interface AzureContainerVersion {
 
 // @internal @deprecated
 export class AzureFunctionTokenProvider implements ITokenProvider {
-    constructor(azFunctionUrl: string, user?: Pick<AzureMember<any>, "additionalDetails" | "userId" | "userName"> | undefined);
+    constructor(azFunctionUrl: string, user?: Pick<AzureMember<any>, "userId" | "userName" | "additionalDetails"> | undefined);
```

Comment:
might be a similar case to mocha-test-setup? Haven't seen this one specifically before (though I vaguely recall a similar api-extractor issue around random reordering along these lines... if CI passes, not a big deal)

Reply:
reverted